tests/e2e/utils/deleteMedia.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Creates a new story.
+ *
+ * @param {string}fileName File name to delete with ext.
+ */
+async function deleteMedia(fileName) {
+  await visitAdminPage('upload.php', 'mode=list');
+
+  await expect(page).toMatch(fileName);
+
+  await expect(page).toClick('a', { text: fileName, exact_text: true });
```

Comment:
Do we have to navigate to the single attachment, can't we use the "Delete" link from the row actions on the list screen?

Something like this perhaps (untested):

```js

// Make row actions appear.
await page.hover(`a[aria-label="“${fileName}” (Edit)"]`)

await Promise.all([
  await expect(page).toClick(`a[aria-label="Delete “${fileName}” permanently"]`),
  page.waitForNavigation()
]);
```

Reply:
I did think about something like this but using a hover this way felt like it might be temperamental. Puppeteer doesn't like clicking links that are not visual. So if the hover did not show the link straight away it might be an issue. Willing to change if I should not worry about these things.

Reply:
Maybe we can give it a try to see how well it works? Would be 1 less navigation, so could speed up the tests a little bit.

Reply:
Just having a quick look, it seems like row only appears if you hover the tr. See https://github.com/WordPress/wordpress-develop/blob/9745a84ce8219f03a01e781f22f1393ef0fb9195/src/wp-admin/css/list-tables.css#L814-L820. 

I am going to see if they is a magical css select for this.

Reply:
This is a work around that is a little dirty - https://github.com/google/web-stories-wp/pull/5849/commits/e0a3871ce2b4c187cfb26d944ab4bfcf0cb77437